distributed/client.py

```diff
@@ -359,6 +358,13 @@ def traceback(self, timeout=None, **kwargs):
     def type(self):
         return self._state.type
 
+    @property
+    def _type(self):
+        try:
+            return self.type.__module__.split(".")[0] + "." + self.type.__name__
+        except AttributeError:
+            return str(self.type)
```

Comment:
There might be a `typename` function somewhere

Reply:
Oooh good point, that's exactly what we want here.